website/addons/osfstorage/model.py

```diff
@@ -218,6 +241,20 @@ def path(self):
     def node(self):
         return self.node_settings.owner
 
+    @property
+    def rented(self):
```

Comment:
Is there any reason this shouldn't return a bool? It seems odd to me to return an empty string, under any circumstances.